src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,119 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.TestService;
+import java.util.ArrayList;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class FirstExercise extends TestService {
+
+    @Test
+    public void exerciseOne() {
+
+        SoftAssertions softly = new SoftAssertions();
+
+        //1. Open test site by URL
+        driver.navigate().to(URL);
+
+        //2. Assert Browser title
+        String actualTitle = driver.getTitle();
+        softly.assertThat(actualTitle).isEqualTo(EXPECTED_TITLE);
+
+        //3. Perform login
+        WebElement signIn = driver.findElement(SIGN_IN_BUTTON);
+        signIn.click();
+
+        WebElement login = webDriverWait
+                .until(ExpectedConditions.visibilityOfElementLocated(LOGIN_INPUT_BUTTON));
+        login.sendKeys(LOGIN_DATA);
+
+        WebElement password = driver.findElement(PASSWORD_INPUT_BUTTON);
+        password.sendKeys(PASSWORD_DATA);
+
+        WebElement enter = driver.findElement(ENTER_BUTTON);
+        enter.click();
+
+        //4. Assert Username is logged in
+        String actualData = driver.findElement(USER_NAME).getText();
+        softly.assertThat(actualData).isEqualTo(EXPECTED_DATA);
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> allActualData = new ArrayList<>();
+
+        allActualData.add(driver.findElement(HOME_BUTTON));
+        allActualData.add(driver.findElement(CONTACT_FORM_BUTTON));
+        allActualData.add(driver.findElement(SERVICE_BUTTON));
+        allActualData.add(driver.findElement(METALS_AND_COLORS_BUTTON));
```

Comment:
Locators could be merged into one, and `driver.findElements` could be used

Reply:
These locators were merged into one and driver.findElements was used.